go.mod

```diff
@@ -6,23 +6,30 @@ go 1.13
 
 require (
 	github.com/Azure/go-autorest/autorest v0.10.0 // indirect
+	github.com/NYTimes/gziphandler v1.0.1 // indirect
```

Comment:
I am just curious why these _indirect_ entries got added, which go version you are using ? can you run _go mod tidy_ if anything changes ? Thanks

Reply:
`go mod tidy` didn't do anything. I think it's a vscode plugin I'm using that is adding these indirect deps. 
I'm running `go version go1.12.15 darwin/amd64`.
I'll reset the file.

Reply:
Oh I see, seems like there is diff in go 1.12 and go 1.13 for go mod :100: 

This project is built and tested with go 1.13, so if possible, you can do development with go 1.13 to avoid any un-intentional surprise :)